omnibus-snaxchain-testnet.toml

```diff
@@ -19,12 +19,12 @@ defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
 
 [var.governance]
 governance_package = "synthetix-governance:3.6.3"
-governance_satellite_package = "synthetix-governance-satellite:3.6.3"
+governance_satellite_package = "synthetix-governance-satellite:3.6.2"
```

Comment:
Why downgrading?

Reply:
this package doesn't exist

Reply:
yeah sure np. I wonder how it passed tests before?